services/approved-studies.js

```diff
@@ -15,14 +16,44 @@ class ApprovedStudiesService {
             console.error(ERROR.APPROVED_STUDIES_INSERTION);
         }
     }
+
+    /**
+     * List Approved Studies API Interface.
+     *
+     * Note:
+     * - This is currently an open API for all logged-in users
+     *   filtering on Organization is not implemented in MVP-2.
+     *
+     * @api
+     * @param {Object} params Endpoint parameters
+     * @param {{ cookie: Object, userInfo: Object }} context request context
+     * @returns {Promise<Object[]>} An array of ApprovedStudies
+     */
+    async listApprovedStudiesAPI(params, context) {
+        verifySession(context)
+          .verifyInitialized();
+
+        return this.listApprovedStudies({});
+    }
+
+    /**
+     * List all approved studies in the collection. Supports filtering.
+     *
+     * @typedef {Object<string, any>} Filters K:V pairs of filters
+     * @param {Filters} [filters] Filters to apply to the query
+     * @returns {Promise<Object[]>} An array of ApprovedStudies
+     */
+    async listApprovedStudies(filters = {}) {
+        return await this.approvedStudiesCollection.aggregate([{ "$match": filters }]);
+    }
 }
 
 class ApprovedStudies {
-    constructor(studyName, studyAbbreviation, dbGaPID, organizationName) {
```

Comment:
Let's correct the spelling here, the correct spelling should be **dbGaPID**. Please correct the spellings in the schema too, thanks!

Reply:
All references to `dbGapID` where just replaced as requested. This did update the `Submission` and `ApprovedStudies` GraphQL types. May effect other PRs.